src/Symfony/Component/Mailer/Bridge/Amazon/Transport/SesHttpAsyncAwsTransport.php

```diff
@@ -87,7 +87,20 @@ protected function getRequest(SentMessage $message): SendEmailRequest
             && $sourceArnHeader = $message->getOriginalMessage()->getHeaders()->get('X-SES-SOURCE-ARN')) {
             $request['FromEmailAddressIdentityArn'] = $sourceArnHeader->getBodyAsString();
         }
+        if (($message->getOriginalMessage() instanceof Message)
+            && $configurationSetHeader =  $message->getOriginalMessage()->getHeaders()->get('X-SES-MESSAGE-TAGS')) {
+            $request['EmailTags'] = $this->arrayifyMessageTags($configurationSetHeader->getBodyAsString());
+        }
 
         return new SendEmailRequest($request);
     }
+
+    protected function arrayifyMessageTags(string $tags): array
+    {
+        return array_map(function (string $tag) {
+            $tag = explode('=', trim($tag), 2);
+
+            return ['Name' => $tag[0], 'Value' => $tag[1]];
+        }, explode(',', $tags));
```

Comment:
shouldn't we care about escaping? 

ie. handling keys/values with special chars like `=` or `,`?

Reply:
How would you approach this?